### PR TITLE
124 update prefix guidance

### LINKED ISSUE #124: Review project identifier prefix guidance for clarity
Review [guidance](http://standard.open-contracting.org/infrastructure/master/en/guidance/identifiers/#globally-unique-project-identifiers) and check that it is clear that prefixes are assigned for series of project identifiers, rather than individual projects.

### PR DESCRIPTION
**Pull request by duncandewhurst**
Closes #124 

Also adds a missing comma to the guidance on project identifiers in OCDS, added in https://github.com/open-contracting/infrastructure/pull/166, to make the meaning clear.

### CODE REVIEW

File: docs/guidance/identifiers.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ There are different approaches to including project identifiers in contracting d
 
 In OCDS, the identifier for the individual infrastructure project to which a contracting process is related should be disclosed using the `planning/project/id` field, introduced in the [Budget and Projects extension](https://extensions.open-contracting.org/en/extensions/budget_project/).
 
-The `planning/budget/projectID` field in OCDS should **not** be used to disclose the identifier for an individual infrastructure project. This field is used to disclose the identifier for a project in the national budget, which might include multiple infrastructure projects to which the contracting process is related.
+The `planning/budget/projectID` field in OCDS should **not** be used to disclose the identifier for an individual infrastructure project. This field is used to disclose the identifier for a project in the national budget, which might include multiple infrastructure projects, to which the contracting process is related.
```
Comment (jpmckinney) [inline docs/guidance/identifiers.md]: I think I am still confused. Can we instead make two sentences from the last sentence?

File: docs/guidance/identifiers.md [hunk 2]
```diff
@@ -43,6 +43,8 @@ If local project identifiers are available in existing systems or data, these sh
 
 To make local project identifiers globally unique for use in OC4IDS, the OC4IDS Helpdesk assigns a project identifier prefix that publishers must add to local project identifiers, using the following identifier structure: `[project identifier prefix]-[local project identifier]`.
 
+Project identifier prefixes establish an identifier series and the same prefix is used for all identifiers in a series. For example, the prefix `oc4ids-qu8r7p`, assigned by the OC4IDS Helpdesk, will be added to all the project identifiers from the SISOCS system developed by CoST Honduras.
```
Comment (jpmckinney) [inline docs/guidance/identifiers.md]: I'll make a commit for you to review, which will be easier than describing the proposed changes.

Comment (jpmckinney) [reply docs/guidance/identifiers.md]: @duncandewhurst I've made the commit. I think each actor's role and actions are clearer. The concept of an 'identifier series' is also clearer in this framing.

File: docs/guidance/identifiers.md (hunk 1 above)
Comment (duncandewhurst) [reply docs/guidance/identifiers.md]: I've split them out in https://github.com/open-contracting/infrastructure/pull/191/commits/fb43a7931ac5bfe52dc75ecd7b43ce110bb25c2e and explained why it is necessary to disclose both identifiers.

File: docs/guidance/identifiers.md (hunk 2 above)
Comment (duncandewhurst) [reply docs/guidance/identifiers.md]: Thanks, I'm happy with your changes.

**Pull request merged**